src/nvim/ex_cmds2.c

```diff
@@ -868,7 +868,7 @@ void set_context_in_profile_cmd(expand_T *xp, char_u *arg)
     return;
   }
 
-  // TODO: expand function names after "func
+  // TODO: expand function names after "func // NOLINT(readability/todo)
```

Comment:
You must have meant blame. I blamed it repeatedly and found that it was introduced at 72cf89bce8e4230dbc161dc5606f48ef9884ba70 by @tarruda.

I'm marking it `TODO(Thiago de Arruda): ...` Is that okay?